src/main/java/com/xiaomi/infra/pegasus/client/PConfigUtil.java

```diff
@@ -77,13 +74,25 @@ public static Properties loadConfiguration(String configPath) throws PException
                 + "but actual configPath is "
                 + configPath);
       }
+      if (stream == null) {
+        throw new PException("config resource not found: " + configPath);
+      }
+      config.load(stream);
       return config;
     } catch (Throwable e) {
       if (e instanceof PException) {
         throw (PException) e;
       } else {
         throw new PException(e);
       }
+    } finally {
+      if (stream != null) {
+        try {
+          stream.close();
+        } catch (Exception ignore) {
```

Comment:
Better output `debug` info？

Reply:
Abnormal shutdown of resources is not the business logic of the system, I think it can be ignored, what do you think?

Reply:
@totalo  You can throw it as a `PException`. Don't ignore any exception if we can do better.

```diff
-  // ignore
+  throw new PException(e);
```

Reply:
Yeh，Got it! Thank you~